iFixit/toolbox
# chore: Update React to v16

## Details
In order for us to upgrade our code base to React v16.4.1, we first have to upgrade its dependencies to versions that support React 16. This pull upgrades Toolbox to React v16 and certain dependencies to versions that support the latest version of React.

## QA
- Checkout this branch on your local machine
- Run `npm install` and possibly `npm update`
- Run `npm start` to view Toolbox
- Look through all our components and ensure that there are no issues
- Done!
